pkg/objstore/swift/swift.go

```diff
@@ -58,13 +69,57 @@ func NewContainer(logger log.Logger, conf []byte) (*Container, error) {
 		Password:         sc.Password,
 		DomainID:         sc.DomainId,
 		DomainName:       sc.DomainName,
-		TenantID:         sc.TenantID,
-		TenantName:       sc.TenantName,
+		TenantID:         sc.ProjectID,
+		TenantName:       sc.ProjectName,
 
 		// Allow Gophercloud to re-authenticate automatically.
 		AllowReauth: true,
 	}
 
+	// The term `tenant` is used in the deprecated OpenStack Identity v2.
+	// In Identity v3 the term `project` is used instead.
+	switch {
```

Comment:
could you also add a couple of examples how or when to configure project id vs project name in the `docs/storage.md` file

Reply:
updated docs. added link to official documentation with examples